ang/contactlayout/contactlayout.js

```diff
@@ -564,29 +614,13 @@
   // Service for loading relationship type options and displaying loading state.
   angular.module('contactlayout')
     .service('contactLayoutRelationshipOptions', function (crmApi4) {
+      var RELATIONSHIP_TYPES = CRM['contactlayout'].relationshipTypes;
```

Comment:
jslint will flag this unnecessary use of bracket notation. You can do 
```suggestion
      var RELATIONSHIP_TYPES = CRM.contactlayout.relationshipTypes;
```